aperture.go

```diff
@@ -180,10 +180,13 @@ func NewAperture(cfg *Config) *Aperture {
 func (a *Aperture) Start(errChan chan error) error {
 	var err error
 
-	// Start the prometheus exporter.
-	if err := StartPrometheusExporter(a.cfg.Prometheus); err != nil {
-		return fmt.Errorf("unable to start the prometheus exporter: "+
-			"%v", err)
+	if a.cfg.Prometheus != nil {
```

Comment:
Should we instead add a `DefaultConfig()` function and use that in `aperture.go#L366`?

Reply:
Can do. Or can add a  "NewConfig" func for now since we dont really have other defaults and just want to initialise the pointer variables. wdyt?